addons/udes_stock/views/stock_picking.xml

```diff
@@ -101,6 +101,15 @@
                 <attribute name="invisible">1</attribute>
             </xpath>
 
+            <!-- Remove Return button for all -->
+            <xpath expr="//button[@name='%(stock.act_stock_return_picking)d']" position="replace"/>
```

Comment:
Can we make this invisible instead of replacing? (Same for others)

It's a bit safer, incase another view references it in future.

`<attribute name="invisible">1</attribute>` should do it.

Reply:
Done